### ISSUE TITLE
Gtk error when running a program => Gtk-CRITICAL **: gtk_text_view_scroll_mark_onscreen

### ISSUE THREAD

**Issue by lglearn** [leksah/leksah#455]
When I build then run the built program, I always get  a Gtk3 error message (usually 5 times) in my console, the Leksah Log panel does not change (the result of the program is not displayed).

[ Note: I'm a beginner with Leksah as well as cabal & stack ]

I'm also unable to debug my program, maybe it's related to this issue, but maybe I did not understand how to do it (setting a breakpoint works, I see the messages in the Log panel).

Command used to run Leksah from a console:
`stack exec --no-ghc-package-path leksah &`

Content of my Main.hs:
```
module Main (
  main
) where
main = putStrLn "Hello World"

```

Content of my HelloWorld.cabal:
```
name: HelloWorld
version: 0.0.1
cabal-version: >= 1.24
build-type: Simple
license: AllRightsReserved

executable HelloWorld
    main-is: Main.hs
    build-depends:
        QuickCheck -any,
        base -any,
        doctest -any
    default-language: Haskell2010
    hs-source-dirs: src
    other-modules:
    ghc-options: -ferror-spans
```

Console message (5 times):

> (leksah:19935): Gtk-CRITICAL **: gtk_text_view_scroll_mark_onscreen: assertion 'get_buffer (text_view) == gtk_text_mark_get_buffer (mark)' failed

BUT, when I restart Leksah, open the workspace and _immediately_ run the program, it works!
Here is the result I see in the _Log panel_:
> ~/haskell/HelloWorld$ ~/haskell/HelloWorld/dist-newstyle/build/HelloWorld-0.0.1/build/HelloWorld/HelloWorld
> Hello World      <==== what I expected here

If I run it again, it works every time.

If it worked and I try again to build then run it, it fails again with the GTK error messages...

In summary, the program can only be launched if no BUILD operation is executed.

**My configuration:**
- Linux Mint 18.2 “Sonya” Cinnamon
- Gtk 3.18.9
- stack:  Version 1.5.1, Git revision 600c1f01435a10d127938709556c1682ecfd694e (4861 commits) x86_64 hpack-0.17.1
- cabal: cabal-install version 1.24.0.2, (the 2.0 is also installed).
- Leksah 0.16.2.2, built from Git, with the Stack method.

I have downloaded & installed _from scratch_ stack, cabal & Leksah yesterday.

Thx.

**Comment by dnovick** [comment]
So, it turns out that I am having exactly the same problem. I just started using Leksah yesterday and ran into this behavior today. It would be great if somebody could address it or provide a workaround. Right now, I end up just running my app from the command line.
